recipes/cofold/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: cofold
+  version: "2.0.4"
+
+source:
+  url: https://github.com/HassanAmr/cofold/archive/21b9be68834a242111e17b549f03f4b3c21ba578.tar.gz
```

Comment:
Please use the original branch and include gcc and llvm as dependency as stated in the readme :)